implementation/map-to-scatter-gather/README.md

```diff
@@ -0,0 +1,118 @@
+
+# Replace Map with Scatter-Gather
+
+This project is the CDK implementation of ['Replace Map with Scatter-Gather'](https://serverlessland.com/refactoring-serverless/map-to-scatter-gather) pattern. It shows how yo can use SNS in CDK to replace AWS Step Functions `Map` state to implement the [scatter-gather](https://www.enterpriseintegrationpatterns.com/patterns/messaging/BroadcastAggregate.html) pattern.
```

Comment:
Detail: Pattern names like Scatter-Gather are initial caps

Reply:
refactored due to change from map to parallel as agreed.